app/models/bulkrax/csv_entry.rb

```diff
@@ -103,7 +103,7 @@ def build_mapping_metadata
 
         data = object_key.present? ? hyrax_record.send(value['object']) : hyrax_record.send(key.to_s)
         if object_key.present?
-          next self.parsed_metadata[key] = '' if data.empty?
+          next if data.empty?
```

Comment:
the code previously was adding numerated headers if the array had more than 1 object, but non numerated headers if the array had 1 object.
e.g. we would get `contributor_family_name`, `contributor_family_name_1`, `contributor_family_name_2`

now we get `contributor_family_name_1`, `contributor_family_name_2`